dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/service/DataAnalysisService.java

```diff
@@ -41,7 +42,7 @@ public interface DataAnalysisService {
      * @param endDate     end date
      * @return task state count data
      */
-    Map<String, Object> countTaskStateByProject(User loginUser, long projectCode, String startDate, String endDate);
+    Result countTaskStateByProject(User loginUser, long projectCode, String startDate, String endDate);
```

Comment:
`Result` should be encapsulated in `Collonter` layer, not `service` layer

Reply:
But in the interface v2(path:src/main/java/org/apache/dolphinscheduler/api/controller/v2), In most cases, Result be encapsulated in service layer

Reply:
We better be able to revisit this

Reply:
there are too many cases about `Result`, we can discuss where to put the `Result` later